src/utils/frame-storage-listener.js

```diff
@@ -13,35 +12,40 @@ class FrameStorageListener {
 
   start(verifier) {
     const storageStrategy = BaseStorage.verify(this.storageStrategy);
-    this._listener = e => {
-      const {data, source, origin} = e;
-      if (typeof data !== 'string') {
+    listenerMessageChannel('data-capsule', onMessage => onMessage(messageHandler));
```

Comment:
why not just `listenerMessageChannel('data-capsule', messageHandler);`?

Reply:
fixed